edb/server/buildmeta.py

```diff
@@ -122,7 +123,10 @@ def hash_dir(dirname, ext, paths):
         hash_dir(dirname, ext, paths)
 
     if extra_files:
-        paths.extend(extra_files)
+        for extra_file in extra_files:
+            if hasattr(extra_file, 'resolve'):
```

Comment:
This isn't going to save you from mypy, I'm afraid.  Use `isinstance(extra_file, pathlib.Path)`

Reply:
Thanks!